lure.go

```diff
@@ -33,9 +33,13 @@ func main() {
 
 	config, err := loadConfig(*confFile)
 	if err != nil {
-		log.Printf("Error Loading Config: %s\n", err)
+		log.Printf("Error Loading Config with path '%s': %s\n", *confFile, err)
 		os.Exit(1)
 	}
+	if os.Getenv("DRY_RUN") == "1" {
+		log.Println("Running in DryRun mode, not doing the pull request nor pushing the changes")
```

Comment:
it seems we have this exact output already in two other places, what's up with that?

Reply:
I wanted to have the output _before_ executing anything. The output is _just_ before a commit action. Meaning that you cannot stop something wrong before it actually does it.